src/test/java/apoc/custom/CypherProceduresTest.java

```diff
@@ -29,7 +30,10 @@
 
     @Before
     public void setUp() throws Exception {
-        db = new TestGraphDatabaseFactory().newImpermanentDatabase();
+        db = new TestGraphDatabaseFactory()
+                .newImpermanentDatabaseBuilder()
+                .setConfig("apoc.custom.procedures.refresh", "100")
```

Comment:
not needed in the cluster tests?

Reply:
We already have it in there:
https://github.com/neo4j-contrib/neo4j-apoc-procedures/blob/3575757f02a7e0adedd422d7c5a20d05edce4f9e/src/test/java/apoc/custom/CypherProceduresClusterTest.java#L37